samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -48,12 +48,23 @@ Configuration
 Angle of arrival mode
 =====================
 
-To build this sample with AoA mode only, set :makevar:`EXTRA_CONF_FILE` to the :file:`overlay-aoa.conf` file using the respective :ref:`CMake option <cmake_options>`.
+This sample builds the angle of arrival (AoA) mode by default.
 
 For more information about configuration files in the |NCS|, see :ref:`app_build_system`.
 
 .. bt_dir_finding_tx_aoa_mode_end
```

Comment:
This block of text is included in the Direction finding peripheral sample, but the new AoD section is not added to it. Is this intentional?

Reply:
This PR is only addressing connectionless tx; a subsequent PR has to be submitted to change implementation in peripheral sample and update documentation accordingly. Yes, for now, it is intentional (to have not updated peripheral sample), and outside the scope of the changes in this PR.

Reply:
You'll need to make a change to the peripheral documentation in this PR, I think, otherwise it will show that AoA is enabled by default when it isn't before the change is implemented.

Reply:
wait... you are referring to connectionless rx ... not peripheral, they are different... sure, I need to check the connectionless rx sample's default mode... putting PR in DNM.

Reply:
Ah, I might've missed which was the sample that pulls the include.

Reply:
i feel the references are ok, the peripheral sample refer to sections correctly in connectionless tx sample.

>$ git grep tx_aoa_
samples/bluetooth/direction_finding_connectionless_tx/README.rst:.. bt_dir_finding_tx_aoa_mode_start
samples/bluetooth/direction_finding_connectionless_tx/README.rst:.. bt_dir_finding_tx_aoa_mode_end
samples/bluetooth/direction_finding_peripheral/README.rst:   :start-after: bt_dir_finding_tx_aoa_mode_start
samples/bluetooth/direction_finding_peripheral/README.rst:   :end-before: bt_dir_finding_tx_aoa_mode_end
$ git grep tx_ant_
samples/bluetooth/direction_finding_connectionless_tx/README.rst:.. bt_dir_finding_tx_ant_aod_start
samples/bluetooth/direction_finding_connectionless_tx/README.rst:.. bt_dir_finding_tx_ant_aod_end
samples/bluetooth/direction_finding_peripheral/README.rst:   :start-after: bt_dir_finding_tx_ant_aod_start
samples/bluetooth/direction_finding_peripheral/README.rst:   :end-before: bt_dir_finding_tx_ant_aod_end

Reply:
> samples/bluetooth/direction_finding_peripheral/README.rst: :start-after:[ bt_dir_finding_tx_aoa_mode_start
samples/bluetooth/direction_finding_peripheral/README.rst: :end-before: bt_dir_finding_tx_aoa_mode_end

This part means that the Peripheral sample's documentation directly copies the text that is between these tags in the Beacon sample. So when this PR gets merged, [this section](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/direction_finding_peripheral/README.html#angle_of_arrival_mode) will change to match the beacon documentation, saying that AoA is enabled by default.

Reply:
@wiba-nordic are you blocking this PR? or can I get this merged? i am confused as to what to change... the other peripheral PR has been merged without documentation updates: https://github.com/nrfconnect/sdk-nrf/pull/17987

@ryanjh FYI... did you not require documentation changes for peripheral sample?

Reply:
Ah, I didn't know it had been changed and merged.